content/en/users/aai/check-in/vos/_index.md

```diff
@@ -284,40 +284,242 @@ From the steps defined [above](#managing-cou-admin-members):
 
 ## VO membership API
 
+### API v2
+
+Check-in provides a REST API that allows clients to manage membership
+information only for the VOs they are authoritative for.
+
+Features:
+
+- Members of the VO are identified via their EGI Check-in Community User Identifier (CUID)
+- Membership can be limited to a specified period
+- All [REFEDS](https://wiki.refeds.org/display/STAN/eduPerson+2020-01#eduPerson202001-eduPersonAffiliation) membership affiliations are supported
+- Role titles are supported
```

Comment:
```suggestion
- All
  [REFEDS](https://wiki.refeds.org/display/STAN/eduPerson+2020-01#eduPerson202001-eduPersonAffiliation)
  membership affiliations are supported
- Role titles are supported
```

Reply:
Please also shorten this line as shown in the example.

Reply:
Done